axelor-stock/src/main/resources/views/MassStockMove.xml

```diff
@@ -0,0 +1,101 @@
+        onClick="save,action-mass-stock-move-method-import-picked-product"
+        help="Allows to add every product from the common stock location"/>
+      <panel-related name="pickedProductListPanel" field="pickedProductList"
+        grid-view="picked-products-mass-stock-move-grid" form-view="picked-products-form"
+        onChange="save" editable="true" colSpan="12"/>
+      <button name="realizePickingBtn" title="Pick all / the rest"
+        onClick="save,action-mass-stock-move-method-realize-picking,action-refresh-record,save"
+        icon="fa-play"/>
+      <button name="cancelPickingBtn" title="Cancel picking"
+        onClick="action-mass-stock-move-method-cancel-picking" icon="fa-undo"/>
+
+    </panel>
+    <panel name="storedProductPanel">
+      <panel-related name="storedProductListPanel" field="storedProductList"
+        grid-view="stored-products-mass-stock-move-grid" form-view="stored-products-form"
+        onChange="save" editable="true" colSpan="12"/>
+      <button name="realizeStorageBtn" title="Store all / the rest"
+        onClick="save,action-mass-stock-move-method-realize-storage"/>
+      <button name="cancelStorageBtn" title="Cancel storage"
+        onClick="save,action-mass-stock-move-method-cancel-storage"/>
+    </panel>
+  </form>
+
+  <action-record name="action-mass-stock-move-record-set-company"
+    model="com.axelor.apps.stock.db.MassStockMove">
+    <field name="company" expr="eval:__user__.activeCompany"
+      if="__user__.activeCompany != null"/>
+    <field name="company" expr="eval:__repo__(Company).all().fetchOne()"
+      if="__user__.activeCompany == null &amp;&amp; __repo__(Company).all().count() == 1"/>
+  </action-record>
+
+  <action-record name="action-mass-stock-move-record-default"
+    model="com.axelor.apps.stock.db.MassStockMove">
+    <field name="cartStockLocation" expr="eval: company.stockConfig?.cartStockLocation"
+      if="company.stockConfig.isMassStockMoveManaged"/>
+  </action-record>
+
+  <action-method name="action-mass-stock-move-method-import-picked-product">
+    <call class="com.axelor.apps.stock.web.MassStockMoveController"
+      method="importProductFromStockLocation"/>
+  </action-method>
+
+  <action-method name="action-mass-stock-move-method-realize-picking">
+    <call class="com.axelor.apps.stock.web.MassStockMoveController" method="realizePicking"/>
+  </action-method>
+
+  <action-method name="action-mass-stock-move-method-cancel-picking">
+    <call class="com.axelor.apps.stock.web.MassStockMoveController" method="cancelPicking"/>
+  </action-method>
+
+  <action-method name="action-mass-stock-move-method-realize-storage">
+    <call class="com.axelor.apps.stock.web.MassStockMoveController" method="realizeStorage"/>
+  </action-method>
+
+  <action-method name="action-mass-stock-move-method-cancel-storage">
+    <call class="com.axelor.apps.stock.web.MassStockMoveController" method="cancelStorage"/>
+  </action-method>
```

Comment:
all these actions methods cause an error when called from the view. They should not be added in this PR